src/cryptonote_core/blockchain.cpp

```diff
@@ -1920,7 +1920,7 @@ bool Blockchain::handle_get_objects(NOTIFY_REQUEST_GET_OBJECTS::request& arg, NO
     if (missed_tx_ids.size() != 0)
     {
       // do not display an error if the peer asked for an unpruned block which we are not meant to have
-      if (tools::has_unpruned_block(get_block_height(bl.second), get_current_blockchain_height(), get_blockchain_pruning_seed()))
+      if (arg.prune || tools::has_unpruned_block(get_block_height(bl.second), get_current_blockchain_height(), get_blockchain_pruning_seed()))
```

Comment:
From buildbot:
```
/Users/buildbot/slave/monero-static-osx-10_11/build/src/cryptonote_core/blockchain.cpp:1923:15: error: no member named 'prune' in 'epee::misc_utils::struct_init<cryptonote::NOTIFY_REQUEST_GET_OBJECTS::request_t>'
      if (arg.prune || tools::has_unpruned_block(get_block_height(bl.second), get_current_blockchain_height(), get_blockchain_pruning_seed()))
```

Reply:
Ah, that relies on some wip code, sorry. No reason to merge to master then, I'll close.